service/src/main/kotlin/fi/espoo/evaka/placement/PlacementPlanService.kt

```diff
@@ -131,8 +132,10 @@ class PlacementPlanService(
                     requestedStartDate?.let { plan.preschoolDaycarePeriod?.copy(start = it) }
                         ?: plan.preschoolDaycarePeriod
                     )!!
-                // TODO: this should not be hard-coded
-                val term = derivePreschoolTerm(period.start)
```

Comment:
I think you might as well replace all usages of this function and just delete it completely.